src/java/org/apache/cassandra/config/Config.java

```diff
@@ -568,6 +568,8 @@ public static void setClientMode(boolean clientMode)
 
     public volatile int consecutive_message_errors_threshold = 1;
 
+    public volatile SubnetGroups client_error_reporting_exclusions = new SubnetGroups();
```

Comment:
Do we need to add `SubnetGroups` to `DatabaseDescriptorRefTest`?

Reply:
Aside: Probably a good idea to make this `volatile` for now even if we don't actually allow modifying it at runtime.

Reply:
`DatabaseDescriptorRefTest` didn't fail... maybe lazy load as init doesn't load this class?

Reply:
heh... locally it fails but it didn't in CI for me... adding